blended-launcher/src/main/runner/bin/launcher.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/sh
 
-TOOLDIR="$(dirname $(readlink -f "$0"))/.."
+OLDDIR=$(pwd)
+TOOLDIR="$(dirname $0)/.."
+cd $TOOLDIR
+export TOOLDIR=$(pwd)
```

Comment:
As we now intent to use this value in config files, we should rename it to something blended specific, like `BLENDED_HOME`. WDYT?